Sming/Services/DateTime/DateTime.cpp

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
For consistency the following methods in the DateTime class should be renamed:

* parseHttpDate(const String& httpDate) -> fromHttpDate(const String& httpDate)
* convertFromUnixTime(time_t timep,..) -> fromFromUnixTime(time_t timep,..) 
* convertToUnixTime(int8_t sec, ..)  -> toUnixTime(int8_t sec, ..)

In addition if the DateTime class should deal with different date formatting then the following method could be a very good addition

`format(const String& format)`

Example:
```
DateTime d;
String httpDate = d.format("D, d M Y H:i:s GMT");
```

Reply:
> Do you want me to rationalise this?

Yes, but that should be done as a separate PR (after this one gets merged). The following refactoring changes should be done:
- Move DateTime from Services to SmingCore.
- Rename the old methods according to the rules above.
- In the DateTime.h header added the old methods that should redirect to the new ones, plus mark them as deprecated. Example:

```C++
class DateTime {
public: 
    /** @brief  Parse a HTTP full date and set time and date
     *  @param  httpDate HTTP full date in RFC 1123 format, e.g. Sun, 06 Nov 1994 08:49:37 GMT
     *  @retval bool True on success
     *  @note   Also supports obsolete RFC 850 date format, e.g. Sunday, 06-Nov-94 08:49:37 GMT where 2 digit year represents range 1970-2069
     *  @note   GMT suffix is optional and is always assumed / ignored
     *
     *   @deprecated Use `fromHttpDate` instead
     */
     bool parseHttpDate(const String& httpDate)   __attribute__ ((deprecated))
     {
            return fromHttpDate(httpDate);
     }
}
```